src/content/about/index.md

```diff
@@ -1,6 +1,6 @@
 ---
 title: About GSA SmartPay®
-description: "The GSA SmartPay program is the world’s largest government charge card and commercial payment solutions program."
```

Comment:
I think deleting the description in this metadata will result in an empty `<meta name="description">` tag on the `/about/` page. This looks like it's not intentional.

Reply:
The description has been added back. Removed by mistake.